v1.2/view-node-details.md

```diff
@@ -71,6 +71,12 @@ Flag | Description
 `--insecure` | Run in insecure mode. If this flag is not set, the `--certs-dir` flag must point to valid certificates.<br><br>**Env Variable:** `COCKROACH_INSECURE`<br>**Default:** `false`
 `--port` | The server port to connect to. <br><br>**Env Variable:** `COCKROACH_PORT`<br>**Default:** `26257`
 
+The `node ls` subcommand also supports the following general flags:
+
+Flag | Description
+-----|------------
+`--timeout` | Set the duration of time that the subcommand is allowed to run before it returns an error and prints partial information. The timeout is specified with a suffix of `s` for seconds, `m` for minutes, and `h` for hours.
```

Comment:
@tschottdorf if the timeout flag isn't set, is there a timeout flag default? Or could the subcommand hang?

Reply:
If there's no timeout, the subcommand may hang.